### PR TITLE
Generate images

### PR DESCRIPTION
**Pull request by rdkdd**
Signed-off-by: Radek Duda <rduda@redhat.com>

### CODE REVIEW

File: spice/deps/helper_cb.py [hunk 1]
```diff
@@ -27,12 +27,12 @@
 import os
 import logging
 import argparse
+from PIL import Image, ImageDraw
```
Comment (Andrei-Stepanov) [inline spice/deps/helper_cb.py]: PEP8 standard. It is a good way to write python code.

https://www.python.org/dev/peps/pep-0008/#imports

Plus

https://google.github.io/styleguide/pyguide.html?showone=Imports#Imports

Use imports for packages and modules only.

Something says me, that Image / ImageDraw are classes.

And

PIL is not standard python library.

from PIL import Image, ImageDraw

ImportError: No module named PIL

File: spice/deps/helper_cb.py [hunk 2]
```diff
@@ -50,12 +50,17 @@
...
 group.add_argument("-d", "--cb2txtf", metavar='FILE',
                    help="Dump text from clipboard to file.")
 group.add_argument("-o", "--cb2stdout", action='store_true',
                    help="Dump clipboard to stdout.")
-group.add_argument("-q", "--query", action='store_true',
-                   help="Dump information about clipboard.")
+group.add_argument("-g", "--genimg", metavar='INT', nargs='?', type=int,
+                   help="Generate an image of side size INT pixels.")
+parser.add_argument("file_n", nargs='?', metavar='FILE', 
+      default="test.png", help="Specify file name.")
```
Comment (Andrei-Stepanov) [inline spice/deps/helper_cb.py]: This line also fails PEP8 standard )
(maybe we should fix Travis check)

File: spice/deps/helper_cb.py [hunk 3]
```diff
@@ -69,6 +74,18 @@
...
+elif args.genimg:
+    size_img = args.genimg
+    img = Image.new('RGB', size=(size_img, size_img), color='yellow')
+    draw = ImageDraw.Draw(img)
+    for i in range(0, size_img, 3):
+        draw.line([(i, 0), (size_img, size_img)], fill='green')
+        draw.line([(0, i), (size_img, size_img)], fill='red')
+        draw.line([(size_img, 0), (0, i)], fill='blue')
+    draw.ellipse((0.6*size_img, 0.15*size_img, 0.8*size_img, 0.3*size_img),
+                                                               fill='red')
```
Comment (Andrei-Stepanov) [inline spice/deps/helper_cb.py]: nice)))

File: spice/deps/helper_cb.py (hunk 1 above)
Comment (rdkdd) [inline spice/deps/helper_cb.py]: No Image and ImageDraw are modules (see http://effbot.org/imagingbook/image.htm, http://effbot.org/imagingbook/image.htm, http://effbot.org/imagingbook/imagedraw.htm). PIL is library. It works on VMs in the bkr02 machine without any additional pip installation

Comment (Andrei-Stepanov) [inline spice/deps/helper_cb.py]: >>> from PIL import Image
Traceback (most recent call last):
  File "<stdin>", line 1, in <module>
ImportError: No module named PIL

This package is provided by RPM: python-pillow
It is necessary to add this RPM name to
tp-spice/requirements-travis.txt
tp-spice/requirements.txt

Yes, it seems that  Image, ImageDraw are modules. (Which violates python naming convention).

Okay, than: https://www.python.org/dev/peps/pep-0008/#imports is still important.

**Pull request merged**